source/_docs/upstream-updates.md

```diff
@@ -165,3 +165,35 @@ There are multiple reasons that 503 errors might occur when updating:
 - PHP segfault: These are tricky to troubleshoot because very little debugging information is present. A temporary fix is available. Contact Pantheon Customer Support if you think you have been affected.
 
 - Timeouts are another cause of 503 errors, though they are much less likely to occur if you are using the Pantheon domains. If the operation takes more than 60 seconds, you might see a timeout occur.
+
+### Process to Resolve Conflicts Locally responds with 'Already-up-to-date'
+If you know your site's core codebase does not match Pantheon's upstream, and attempts to resolve it locally do not work because your site repository is seen as up-to-date, you may need to manually overwrite core files. We only recommend this method as an absolute last resort.
+
+1. Download an archive of the Pantheon upstream:
+
+    * <a href="https://github.com/pantheon-systems/WordPress/archive/master.zip">WordPress <span class="glyphicons glyphicons-download-alt"></span></a>
+    * <a href="https://github.com/pantheon-systems/drops-8/archive/master.zip">Drupal 8 <span class="glyphicons glyphicons-download-alt"></span></a>
+    * <a href="https://github.com/pantheon-systems/drops-7/archive/master.zip)">Drupal 7 <span class="glyphicons glyphicons-download-alt"></span></a>
+
+2. If you haven't done so already, [clone your Pantheon site repository](/docs/git/#clone-your-site-codebase) using [Terminus](/docs/terminus) (replace `<site>`):
+
+        `terminus connection:info <site>.dev --fields='Git Command' --format=string`
+
+3. Checkout a new branch (optional):
+
+        git checkout -b updates
+
+4. Use a file manager, such as [Finder for MacOS](https://support.apple.com/en-us/HT201732) or [File Explorer for Windows](https://support.microsoft.com/en-us/help/17217/windows-10-whats-changed-in-file-explorer), to copy the contents of the downloaded archive to your site repository excluding paths where custom code is set.
+
+  Exclude at least the following directories when copying, in addition to any other paths your site uses to store custom code:
+
+    * Wordpress: `/wp-content` and `wp-config.php`
+    * Drupal 8: `/sites` and `/modules`
+    * Drupal 7: `/sites`
+
+5. Stage, commit, and push your changes to Pantheon:

```

Comment:
It may be worth calling out a review of the changes as an additional step here. That might be out of scope though so not sure.